index.js

```diff
@@ -0,0 +1,282 @@
+
+         currentScore += Number(getCard.value)
+
+         //add the sum to the current total
+         playerCurr.innerText = `Current Player score is ${currentScore}`;
+         playerDiv.appendChild(playerCurr);
+
+
+         if (currentScore === 21) {
+            playerCurr = document.querySelector("#playerData");
+            playerCurr.innerText = `Player score is ${currentScore}, BLACKJACK`;
+            declare.innerText = "PLAYER WINS!!!";
+
+            startBtn = document.querySelector('#start');
+            startBtn.style.visibility = "visible";
+            hitBtn = document.querySelector("#hit")
+            hitBtn.style.visibility = "hidden";
+            stayBtn = document.querySelector('#stay')
+            stayBtn.style.visibility = "hidden";
+         }
+
+         if (currentScore > 21) {
+            playerCurr = document.querySelector("#playerData")
+            playerCurr.innerText = `Player score is ${currentScore}, you BUSTED`;
+            declare.innerText = "PLAYER LOSE!!!";
+
+            startBtn = document.querySelector('#start');
+            startBtn.style.visibility = "visible";
+            hitBtn = document.querySelector("#hit");
+            hitBtn.style.visibility = "hidden";  
+            stayBtn = document.querySelector('#stay')
+            stayBtn.style.visibility = "hidden";       
+         }
+      }
+
+      catch (err) {
+         console.log(err)
+      }
+   }
+
+   const computerDraws = async () => {
+
+      try {
+         let getThreeCards = await axios.get(`https://deckofcardsapi.com/api/deck/${deck_ID}/draw/?count=3`)
+         let cards = getThreeCards.data.cards
+
+         //cardsImage
+         let ul = document.createElement('ul')
+         let computerDiv = document.createElement('div')
+         computerDiv.id = "computerDiv"
+         let displayCards = document.querySelector('#board')
+         pcCurr = document.createElement('h3')
+         pcCurr.id = "computerData"
+         computerDiv.appendChild(ul)
+         displayCards.appendChild(computerDiv)
+         cards.forEach(card => {
+            let li = document.createElement('li')
+            let cardImgs = document.createElement('img')
+            cardImgs.src = card.image
+            li.appendChild(cardImgs)
+            ul.appendChild(li)
+            if (card.value === "JACK" || card.value === "QUEEN" || card.value === "KING") {
+               card.value = 10
+            } else if (card.value === "ACE") card.value = 11;
+            // console.log(card.value)
+            computerScore += Number(card.value)
+         })
+
+         //display the sum of the 3 current Cards
+         pcCurr.innerText = `Dealer score is ${computerScore}`
+         computerDiv.appendChild(pcCurr)
+
+         //
+         if (computerScore === 21) {
+            startBtn = document.querySelector('#start')
+            startBtn.style.visibility = "visible"
+
+            pcCurr = document.querySelector("#computerData")
```

Comment:
You can make lines 210 -217 a function since it seems that you're doing almost the same thing three times.